helm/values.yaml

```diff
@@ -60,10 +60,11 @@ serviceSelectorLabels:
 configBootstrapperConfig:
   name: config-bootstrapper-config
 
-job:
-  prefix: default
-
-configurationCommands: {}
+jobs:
+  default:
+    prefix: default
```

Comment:
Question: 
Do we need this `prefix` again? Should the key above match this always?
If yes, does it make sense to completely get rid of it?